xoinvader/tests/test_background.py

```diff
@@ -0,0 +1,128 @@
+import pytest
+
+from xoinvader.background import Background, Chunk, load_chunks
+from xoinvader.common import Settings
+from xoinvader.utils import Point, Surface
+
+from copy import copy
+
+def test_chunk():
+    c = Chunk("test")
+    assert not c.lines
+    assert not c.length
+    assert c.name == "test"
+
+    c.add_line("wut")
+    assert c.lines[0] == "wut"
+    assert c.length == 1
+
+    c[0] = "ugh"
+    assert c[0] == "ugh"
+
+def test_load_chunks():
+    with pytest.raises(ValueError):
+        load_chunks("xoinvader/tests/chunk_no_name.bg")
+
+    with pytest.raises(ValueError):
+        load_chunks("xoinvader/tests/chunk_duplicate_names.bg")
+
+    with pytest.raises(ValueError):
+        load_chunks("xoinvader/tests/no_chunks.bg")
+
+    c = load_chunks("xoinvader/tests/chunk_normal.bg")
+    assert(len(c) == 3)
```

Comment:
assert is keyword, using parens can cause triky things, like creating tuples inplace)